### PR TITLE
Staging -> Main

### PR DESCRIPTION
**Pull request by n-young**

### CODE REVIEW

File: src/components/queue/QueueList/QueueList.tsx [hunk 1]
```diff
@@ -63,7 +62,7 @@ const QueueList: FC<QueueListProps> = ({queue, showCompletedTickets}) => {
                 <Typography variant="h6" fontWeight={600}>
                     Queue
                 </Typography>
-                {!queue.isCutOff && !queueEnded && !inQueue && !isTA(queue.course.id) &&
+                {!queue.isCutOff && !queueEnded && !inQueue && //!isTA(queue.course.id) &&
```
Comment (n-young) [inline src/components/queue/QueueList/QueueList.tsx:65]: Will remove this; don't merge yet

File: src/components/shared/AboutDialog/AboutDialog.tsx [hunk 2]
```diff
@@ -21,10 +24,13 @@ const AboutDialog: FC<AboutDialogProps> = ({open, onClose}) => {
...
                 </Typography>
-                {/*<Button startIcon={<GitHubIcon/>} color="inherit" variant="outlined"*/}
-                {/*        href="https://github.com/nthnluu/hours-frontend/wiki/Contributing">*/}
-                {/*    Contribute*/}
-                {/*</Button>*/}
+                <Button startIcon={<GitHubIcon/>} color="inherit" variant="outlined" href={"https://github.com/nthnluu/hours-frontend/wiki/Contributing"}>
+                   Contribute
+                </Button>
+                <Divider/>
+                <Typography>
```
Comment (neilramaswamy) [inline src/components/shared/AboutDialog/AboutDialog.tsx:31]: Can we hyperlink this to https://cs.brown.edu/people/memoriam/william-povell/ as is done in SignMeUp right now?

File: src/components/shared/AccountMenu/AccountMenu.tsx [hunk 3]
```diff
@@ -137,8 +137,7 @@ const AccountMenu: FC<AccountMenuProps> = ({user}) => {
                                         </ListItemButton>
                                     </ListItem>
                                     <ListItem disablePadding>
-                                        <ListItemButton
-                                            onClick={() => handleClick("https://forms.gle/Ue9wQDXDuczLk7b56")}>
+                                        <ListItemButton onClick={() => handleClick("https://forms.gle/Ue9wQDXDuczLk7b56")}>
```
Comment (neilramaswamy) [inline src/components/shared/AccountMenu/AccountMenu.tsx:140]: Will this open on a new tab? Probably better for accessibility as well if we use an `a` or any wrapper component thereof.

File: src/util/auth/api.ts [hunk 4]
```diff
@@ -46,6 +46,7 @@ export interface Notification {
     Body: string;
     Timestamp: Timestamp;
     Type: NotificationType;
+    Data: string;
```
Comment (neilramaswamy) [inline src/util/auth/api.ts:49]: I presume the backend supports this already?

File: src/components/settings/AllCoursesSection/AllCoursesSection.tsx [hunk 5]
```diff
@@ -1,6 +1,6 @@
 import React, {FC, useState} from "react";
 import CreateCourseDialog from "@components/settings/CreateCourseDialog";
-import {List, Typography} from "@mui/material";
+import {List, Stack, Typography, TextField} from "@mui/material";
```
Comment (neilramaswamy) [inline src/components/settings/AllCoursesSection/AllCoursesSection.tsx]: Why are we adding things here if we aren't using them later in the file?

File: src/components/queue/QueueList/QueueList.tsx [hunk 6]
```diff
@@ -9,22 +9,21 @@ import {
...
+    queue: Queue;
+    tickets: Ticket[] | undefined;
+    ticketsLoading: boolean;
 }
 
 /**
  * QueueList lists out the tickets in a queue.
  */
-const QueueList: FC<QueueListProps> = ({queue, showCompletedTickets}) => {
+const QueueList: FC<QueueListProps> = ({queue, tickets, ticketsLoading}) => {
```
Comment (neilramaswamy) [inline src/components/queue/QueueList/QueueList.tsx:25]: Where is `ticketsLoading` being used?

**Review by neilramaswamy — APPROVED** [review]

**Pull request closed without merging**